sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_polling.py

```diff
@@ -135,7 +135,7 @@ def from_continuation_token(
 class DocumentModelAdministrationPolling(OperationResourcePolling):
     """Polling method overrides for training endpoints."""
 
-    def get_final_get_url(self, pipeline_response: PipelineResponseType) -> None:
+    def get_final_get_url(self, pipeline_response: Any) -> None:
```

Comment:
This may be controversial @kristapratico ? The rational is that this method do not use the input at all, so any type would work. The exact type shouldn't matter here, until someone changes this implementation to actually use the response.

You may say I'm cheating, but it's not false either I believe :)

Reply:
Yeah no risk here / totally fair